### ISSUE TITLE
Intel MKL FATAL ERROR: Error on loading function mkl_blas_avx_xdcopy

### ISSUE THREAD

**Issue by dickreuter** [pyinstaller/pyinstaller#2175]
Pyinstaller on mac compiles correctly but when running the file an error is produced:
Intel MKL FATAL ERROR: Error on loading function mkl_blas_avx_xdcopy

To be fair I have to say that the program was compiled on a mackintosh running in a virtual machine on a windows PC. But not sure this is relevant.

**Comment by stephenrauch** [comment]
Has this been handled by #2111

**Comment by dickreuter** [comment]
This error occurs with the latest development branch so I don't think this has been fully resolved.

**Comment by dickreuter** [comment]
The solution is 
conda install nomkl numpy scipy scikit-learn numexpr
conda remove mkl mkl-service

The problem is related to anaconda.

**Comment by Mottotime** [comment]
Have a look at BVLC/caffe#3884

**Comment by RuochenZhao** [comment]
The above fixed my solution while nothing else works! Thanks !!! Lifesaver

**Comment by MarinerT** [comment]
I had the error: 
Intel MKL FATAL ERROR: Error on loading function mkl_blas_avx2_xsdot

Using ' conda update --all' solved it for me.

**Comment by iamzhangyunlu** [comment]
> I had the error:
> Intel MKL FATAL ERROR: Error on loading function mkl_blas_avx2_xsdot
> 
> Using ' conda update --all' solved it for me.

Same problem here, I update all but the problem is still here

**Comment by iamzhangyunlu** [comment]
> I had the error:
> Intel MKL FATAL ERROR: Error on loading function mkl_blas_avx2_xsdot
> 
> Using ' conda update --all' solved it for me.

Same problem here, I update all but the problem is still here @MarinerT

**Comment by MarinerT** [comment]
> > I had the error:
> > Intel MKL FATAL ERROR: Error on loading function mkl_blas_avx2_xsdot
> > Using ' conda update --all' solved it for me.
> 
> Same problem here, I update all but the problem is still here @MarinerT

@iamzhangyunlu Are you able to access packages or other programs in Anaconda? For me the all packages became inaccessible, in Jupyter Notebooks, Spyder, etc. I could use the programs, but I couldn't import packages like NumPy. I forget the steps I took before conda update --all, which obviously are key to have solved it, but they weren't anything fancy.  Knowing me, I probably tried checking if packages exist, like NumPy, then reinstalling it, and using the errors as clues to what to research next. I eventually came to update the whole thing.

As stupid as it sounds, have you tried restarting your computer?
